### ISSUE TITLE
General To Do

### ISSUE THREAD

**Issue by katieb1** [katieb1/NEvolve#4]
- [ ] Profile multi-GPU pandas with 100K dataset (in progress)
- [ ] Profile multi-GPU cudf with 100K dataset
- [ ] Test if Matt's virtual environment with rapids and pytorch will work
- [ ] Build singularity with Matt's git repo
- [ ] Hyperparameter tuning job submission script
- [ ] General research into DDP
- [ ] Train multi-GPU with or without pandas on 1K dataset and make sure nn is learning
- [ ] Run more epochs + 100K dataset through nn training to see if accuracy improves
- [ ] Find baseline hyperparameter values for tuning (perhaps using weights and biases nvidia module `wandb`)
- [ ] Try network w/ and w/o position data to see if accuracy improves
- [ ] Look into binary formats - e.g. hdf5
- [ ] Visualization code
